python/social_bot/envs/play_ground.py

```diff
@@ -67,103 +68,101 @@ class GroceryGround(GazeboEnvBase):
     """
 
     def __init__(self,
+                 agent_type='pioneer2dx_noplugin',
+                 world_name="play_ground.world",
+                 task=GoalWithDistractionTask,
+                 secondary_task=None,
                  with_language=False,
                  use_image_observation=False,
                  image_with_internal_states=False,
-                 task_name='goal',
-                 agent_type='pioneer2dx_noplugin',
                  world_time_precision=None,
                  step_time=0.1,
```

Comment:
world_time_precision is the substep time. it is read from agent.cfg by default. 

step_time=0.1 by default suites for most agent/task (10Hz control rate). But tasks need higher control frequency prefer lower step_time. e.g., iCub is trained with 0.05,  a steptime of 0.1 on the icub walking task won't be success).